src/main/scala/com/cloudant/clouseau/Utils.scala

```diff
@@ -29,4 +35,33 @@ object Utils {
     new BytesRef(string)
   }
 
+  def rename(rootDir: File, dbName: String, sig: String) {
+    val logger = Logger.getLogger("clouseau.utils")
```

Comment:
the rootDir here is always the same, so don't pass it in, simply fetch it within this method. This will make it clearer which directory we are renaming.

Reply:
It looks that there is no`ctx` in Utils.  So I keep this for now.